tests/comparisons/Template/testBakeView.php

```diff
@@ -1,3 +1,9 @@
+<?php
+/**
+ * @var \TestApp\View\AppView $this
+ * @var \Cake\Datasource\EntityInterface $article
```

Comment:
Does bake use the concrete entity if available?
If not no big deal, I think IdeHelper will auto switch them out once they exist anyway :)

Reply:
It uses * @var \{{ entityClass }} ${{ singularVar }} to bake the template so its defined by the bake command

Reply:
The tests case doesn't have a entity so I think it's defaulting to \Cake\Datasource\EntityInterface